internal/cluster.go

```diff
@@ -361,10 +358,19 @@ func (cs *clusterService) GetMemberByUUID(uuid string) core.Member {
 	return nil
 }
 
+func (cs *clusterService) getOwnerConnectionAddress() *proto.Address {
+	address, ok := cs.ownerConnectionAddress.Load().(*proto.Address)
```

Comment:
`return  cs.ownerConnectionAddress.Load().(*proto.Address)` is enough I think.

Reply:
if you omit error parameter, when address is null, cast fails with panic.
It says that nil can not be cast to (*proto.Address)